R/teal_slice.R

```diff
@@ -55,18 +55,18 @@
 #'  Must be a valid `R` expression which can be evaluated in the context of the data set.
 #'  For a `data.frame` `var == "x"` is sufficient, but `MultiAssayExperiment::subsetByColData`
 #'  requires `dataname` prefix, *e.g.* `data$var == "x"`.
-#' @param choices (optional `vector`) specifying allowed choices;
+#' @param choices (`vector`) optional, specifying allowed choices;
```

Comment:
This lost the meaning

> vector optional, specifying allowed choices;

?

We need to come up with something better in here.